python/lsst/ap/pipe/ap_pipe.py

```diff
@@ -541,6 +552,42 @@ def _doIngestCalibs(repo, calib_repo, calib_datafiles, defectfiles):
     return calibingest_metadata
 
 
+def _doIngestTemplates(repo, templateRepo, inputTemplates):
+    '''Ingest templates into the input repository, so that
+    GetCoaddAsTemplateTask can find them.
+
+    After this method returns, butler queries against `templateRepo` can find the
+    templates in `inputTemplates`.
+
+    Parameters
+    ----------
+    repo: `str`
+        The output repository location on disk where ingested raw images live.
+    templateRepo: `str`
+        The output repository location on disk where ingested templates live.
+    inputTemplates: `str`
+        The input repository location where templates have been previously computed.
+
+    Returns
+    -------
+    calibingest_metadata: `PropertySet` or None
+        Metadata from any tasks run by this method
+    '''
+    log = lsst.log.Log.getLogger('ap.pipe._doIngestTemplates')
+    if os.path.exists(os.path.join(templateRepo, 'deepCoadd')):
```

Comment:
Please make a TODO here that `deepCoadd' should be adjusted later: these are actually good seeing coadds, for instance.

Reply:
Will do, although there's a ticket open ([DM-12557](https://jira.lsstcorp.org/browse/DM-12557)) to rework the short-circuiting system entirely.

Reply:
Is there a ticket for changing the directory? For example, are you planning to do it as part of [DM-10978](https://jira.lsstcorp.org/browse/DM-10978)?

Reply:
I think we're still waiting on an RFC (and implementation) of the `coaddName` bit here: https://community.lsst.org/t/coaddition-tasks-redesign-rfd-request/2101/2